subsys/nfc/Kconfig.nfc

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NFCT_IRQ_PRIORITY
+	int
+	prompt "Interrupt priority"
+	range 0 6
+	default 1
+	help
+	  Sets NFC interrupt priority.
+	  Levels are from 0 (highest priority) to 6 (lowest priority)
```

Comment:
Keep in mind this PR: https://github.com/NordicPlayground/nrfxlib/pull/103/files

```default 5 if ZERO_LATENCY_IRQS``` line seems to be missing.

Are you planning on keeping the priority level set to 1? Or are you still testing?

Reply:
I would like to try keeping it set to 1(the same priority like clock).